src/settings/RageshakeButton.module.css

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.rageshakeControl {
+  height: 50px;
+  width: 176px;
```

Comment:
I worry that hardcoding the dimensions like this is not a good idea for i18n

Reply:
Yeah, fair. I've just made it text which means the button changes size causing some jumping, but avoids the hardcoding.